src/backend/distributed/master/master_modify_multiple_shards.c

```diff
@@ -149,9 +149,9 @@ master_modify_multiple_shards(PG_FUNCTION_ARGS)
 
 	if (modifyQuery->commandType != CMD_UTILITY)
 	{
-		bool multiShardQuery = true;
+		bool multiShardQuery = false;
```

Comment:
why this change? seems a bit confusing

Reply:
I know, but this line makes no sense to me. Why can't `multiShardQuery` be true?

Reply:
Otherwise, we need to have planner restriction context for multi shard queries (not multi table) sent by `master_modify_multiple_shards`. We can remain it as it is using the last commit I have sent.